src/ray/rpc/grpc_client.h

```diff
@@ -84,13 +85,16 @@ class GrpcClient {
   /// `FooService::Stub::PrepareAsyncBar` function.
   /// \param[in] request The request message.
   /// \param[in] callback The callback function that handles reply.
+  /// \param[in] call_name The name of the gRPC method call.
+  /// \param[in] method_timeout_ms The timeout of the RPC method in ms.
+  /// -1 means it will use the default timeout configured for the handler.
   ///
   /// \return Status.
   template <class Request, class Reply>
   void CallMethod(
       const PrepareAsyncFunction<GrpcService, Request, Reply> prepare_async_function,
       const Request &request, const ClientCallback<Reply> &callback,
-      std::string call_name = "UNKNOWN_RPC") {
+      std::string call_name = "UNKNOWN_RPC", int64_t method_timeout_ms = -1) {
```

Comment:
where is `method_timeout_ms` used in this method? I can't find any.

Reply:
oh yeah good catch. Let me add them properly. Will add an unit test too